### PR TITLE
Update Camel-forge addon repository

### PR DESCRIPTION
**Pull request by oscerd**
Hi @gastaldi ,

Just a little update in docs. The camel-forge addon has a new repository now: https://github.com/fabric8io/fabric8-forge

Thank you,
Andrea

### CODE REVIEW

File: addons-community.yaml [hunk 1]
```diff
@@ -257,7 +257,7 @@ description: Setup/Manage Apache Camel in your project
 url: http://fabric8.io/guide/forge.html
 website: http://fabric8.io/guide/forge.html
 installCmd: addon-install --coordinate io.fabric8.forge:camel
-repo: https://github.com/fabric8io/fabric8.git
+repo: https://github.com/fabric8io/fabric8-forge.git
 path: /forge/addons/camel
```
Comment (gastaldi) [inline addons-community.yaml]: Path needs to be changed to /addons/camel, since /forge no longer exists in this repository

**Comment by oscerd** [comment]
Good catch. Sorry about this.

**Comment by gastaldi** [comment]
Fantastic. Thank you so much!

**Comment by oscerd** [comment]
In the next days, I will submit some PRs related to the other addons from @jstrachan, @davsclaus and the other @fabric8io guys

**Comment by gastaldi** [comment]
Looking forward to it! You guys rock

**Pull request merged**